frontend/src/RecipeCard/RecipeCard.jsx

```diff
@@ -0,0 +1,35 @@
+import './RecipeCard.css'
+import React, { useState } from "react";
+import filler_img from '../assets/placeholder_img.jpeg'
+
+const RecipeCard = (props) => {
+    const [isFav, setIsFav] = useState(false);
+
+    return (
+
+        <div className="recipe-card" onClick={props.handleClickFunc}>
+            <section>
+                <img className ="image" src= {filler_img} />
+                <h2 className = "recipeName" > Recipe Name </h2>
+                <div className='UnderName'>
```

Comment:
for consistency - can we keep all the classNames in camelCase?

Reply:
Okay, I'll keep them all in camelCase.